xrootdlib/structs/XrdXrootdMon/plugin.py

```diff
@@ -0,0 +1,74 @@
+import json
+import enum
+from typing import Union
+
+from ...utility import slot_repr
+
+
+class pluginType(str, enum.Enum):
+    """
+    Assumes binary header.
+    """
+    # Cache Context Manager 
+    isCCM = 'M'
+
+    # Proxy File Cache 
+    isPFC = 'C'
+
+    # TCP Connection Monitor 
+    isTCM = 'T'
```

Comment:
While I'd argue that it doesn't really matter since one is a superset of the other I'll go with bytes for clarity's sake. Thanks for the suggestion :+1: